video/out/vulkan/context_display.c

```diff
@@ -424,6 +424,14 @@ static bool display_init(struct ra_ctx *ctx)
         .alphaMode = VK_DISPLAY_PLANE_ALPHA_OPAQUE_BIT_KHR,
     };
 
+    PFN_vkCreateDisplayPlaneSurfaceKHR vkCreateDisplayPlaneSurfaceKHR =
```

Comment:
you probably don't want to alias the symbol name due to `#define`'s or whatnot

Reply:
I usually just drop the `vk` prefix for this reason.